assets/src/edit-story/components/library/panes/media/common/paginatedMediaGallery.js

```diff
@@ -108,6 +110,11 @@ function PaginatedMediaGallery({
     }
   }, [hasMore, isMediaLoaded, isMediaLoading, setNextPage]);
 
+  // Scroll to the top when the searchTerm or selected category changes.
+  useEffect(() => {
```

Comment:
is it possible to test this in karma?

Reply:
Added a test for when a category is selected.